protostar/starknet_gateway/multicall/multicall_file_example.py

```diff
@@ -0,0 +1,23 @@
+from typing import Union
+
+MULTICALL_FILE_EXAMPLE = """[[call]]
+# defines an contract address identifier
+id = "my_contract"
+type = "deploy"
+class-hash = CONTRACT_CLASS_HASH
+inputs = []
+
+[[call]]
+type = "invoke"
+function = "increase_balance"
+
+# contract-address accepts a contract address or an identifier
+contract-address = "$my_contract"
+
+# calldata accepts felts or identifiers
+inputs = [42]
```

Comment:
Is data transformer support considered? I think inputs might be misleading as opposed to `calldata`. On the other hand `calldata` does not contain identifiers. WDYT?

Reply:
I think we should be consistent with CLI. Data transformer can always be added later.

Reply:
CLI is accepting inputs but is accepting data transformer, thus the logic is not consistent - right?

Reply:
Support for data transformer could be added here as well.